spec/rubocop/cop/rspec/describe_class_spec.rb

```diff
@@ -51,6 +51,33 @@
     RUBY
   end
 
+  context 'with non-class constants' do
+    it 'ignores them' do
+      expect_no_offenses(<<-RUBY)
+        describe 'Some::VERSION' do
+          subject { Object.const_get(self.class.description) }
+        end
+      RUBY
+    end
+
+    it 'ignores root-constants with `::` at start' do
```

Comment:
"ignores top-level constants"

Reply:
> "ignores top-level constants"

Replaced "root-constants" with "top-level constants", but they may be referenced without `::` at the start, so I left this.